packages/examples/dashboard/src/components/Dashboard/Dashboard.tsx

```diff
@@ -57,23 +49,23 @@ export default function App() {
             <div className="px-6 py-3 w-full flex justify-end">
               <SimpleFilter
                 query={{
-                  columnName: process.env.REACT_APP_DIMENSION_1,
-                  dataPool: { name: process.env.REACT_APP_DATA_POOL_UNIQUE_NAME_1 },
+                  columnName: envs.REACT_APP_DIMENSION_1,
+                  dataPool: { name: envs.REACT_APP_DATA_POOL_UNIQUE_NAME_1 ?? '' },
                   maxValues: 1000,
                   timeRange: { relative: RelativeTimeRange.LastNDays, n: 30 }
                 }}
                 autocompleteProps={{ containerStyle: { width: '500px' }, placeholder: 'Filter by taco name' }}
               />
             </div>
             <div className="grid grid-cols-2 gap-2">
-              <TimeSeriesStaticTest />
-              <TimeSeriesConnectedTest />
-              <LeaderboardStaticTest />
-              <LeaderboardConnectedTest />
-              <CounterStaticTest />
-              <CounterConnectedTest />
-              <PieChartStaticTest />
-              <PieChartConnectedTest />
+              <TimeSeriesStatic />
+              <TimeSeriesConnected envs={envs} />
```

Comment:
Maybe we can receive `envs` once and then override `process.env` so we don't prop drill it?

Reply:
We may have something in `process.env` later. Let's see how it goes.